src/ucs/datastruct/static_bitmap.h

```diff
@@ -135,6 +135,242 @@ BEGIN_C_DECLS
     UCS_STATIC_BITMAP_FUNC_CALL(UCS_PP_UNIQUE_ID, _bitmap, \
                                 ucs_bitmap_bits_is_zero)
 
+
+/**
+ * Find first set bit in the bitmap. If the bitmap is all zero, the result is
```

Comment:
Q: why the result is undefined?
Shouldn't it be just the index above bitmap size?